plugins/modules/zos_fetch.py

```diff
@@ -446,7 +446,7 @@ def _fetch_pdse(self, src, is_binary, encoding=None):
             )
         if (not is_binary) and encoding:
             enc_utils = encode.EncodeUtils()
-            from_code_set = encoding.get("from")
+            from_code_set = encode.Defaults.DEFAULT_MVS_CHARSET
```

Comment:
Quick question, does this mean that the "from" in this case will never be honored and only the default will be used?

Reply:
would something like `from_code_set = encoding.get("from", encode.Defaults.DEFAULT_MVS_CHARSET)` make more sense?

Reply:
Good question! Yes that would make more sense. Although MVS data sets are almost always encoded in IBM-037.